charts/kyverno/templates/configmap.yaml

```diff
@@ -9,16 +9,18 @@ metadata:
 data:
   # resource types to be skipped by kyverno policy engine
   {{- if .Values.config.resourceFilters }}
-  resourceFilters: {{ tpl (join "" .Values.config.resourceFilters) . | quote }}
+  resourceFilters: {{ include "kyverno.resourceFilters" . | quote }}
   {{- end -}}
   {{- if .Values.config.excludeGroupRole }}
   excludeGroupRole: {{ join "," .Values.config.excludeGroupRole | quote }}
   {{- end -}}
   {{- if .Values.config.excludeUsername }}
   excludeUsername: {{ join "," .Values.config.excludeUsername | quote }}
   {{- end -}}
-  {{- if .Values.config.webhooks }}
-  webhooks: {{ .Values.config.webhooks  | toJson | quote }}
+  {{- if or .Values.config.webhooks }}
+  webhooks: {{ include "kyverno.webhooks" . | quote }}
+  {{- else if .Values.excludeKyvernoNamespace }}
+  webhooks: '[{"namespaceSelector": {"matchExpressions": [{"key":"kubernetes.io/metadata.name","operator":"NotIn","values":["{{ include "kyverno.namespace" . }}"]}]}}]'
```

Comment:
Would be great to have this managed inside the template so that we have the logic at a single place.

Reply:
Hmmm, maybe you’re right, we potentially need to ignore the key entirely :(